go.mod

```diff
@@ -11,7 +11,7 @@ require (
 	github.com/elastic/go-elasticsearch/v7 v7.14.0
 	github.com/elastic/go-licenser v0.3.1
 	github.com/elastic/go-ucfg v0.8.3
-	github.com/elastic/package-spec/code/go v0.0.0-20210811110254-a7173524bc76
+	github.com/elastic/package-spec v1.0.1-0.20210831025616-dc22ddb86159
```

Comment:
Umm, this tag doesn't exist yet. I can create a new release tomorrow.

Reply:
No tag needed just yet, since this is using the commit hash.